### PR TITLE
ci: Drop manual checkout of merge commit

### PR DESCRIPTION
**Pull request by real-or-random**
This is no longer necessary as of
https://github.com/cirruslabs/cirrus-ci-docs/issues/791#issuecomment-1615691585 .

### CODE REVIEW

**Comment by real-or-random** [comment]
cc @MarcoFalke

**Comment by real-or-random** [comment]
The merge commit is correctly checked out, see https://cirrus-ci.com/task/5333035701239808?logs=test#L2 in the CI logs.

**Comment by jonasnick** [comment]
Where does one set the "config resolution strategy"?

**Review by jonasnick — APPROVED** [review]
Thanks @MarcoFalke.

ACK 98579e297b19bbb23c924f21942eccdbd618de67

**Review by alokeutpal — APPROVED** [review]

File: ci/cirrus.sh [hunk 1]
```diff
@@ -4,7 +4,8 @@ set -eux
 
 export LC_ALL=C
 
-# Print relevant CI environment to allow reproducing the job outside of CI.
+# Print commit and relevant CI environment to allow reproducing the job outside of CI.
+git show --no-patch
```
Comment (hebasto) [inline ci/cirrus.sh:8]: It would be nice to have a similar command in the native Windows task as well.

**Review by hebasto — COMMENTED** [review]
Post-merge ACK 98579e297b19bbb23c924f21942eccdbd618de67.

**Pull request merged**